webui/react/src/components/UPlot/UPlotChart.tsx

```diff
@@ -26,141 +25,208 @@ interface ScaleZoomData {
   min?: number;
 }
 
+const shouldUpdate = (
```

Comment:
**nit:** Side note: I prefer `shouldRecreate` to differentiate updates from recreates, where updates to an existing instance of a chart is separate from the recreation of the chart where the end result is a new instance of a chart.

Reply:
👍